hawkbit-dmf-amqp/src/main/java/org/eclipse/hawkbit/amqp/AmqpConfiguration.java

```diff
@@ -58,6 +60,9 @@
     @Autowired
     private ConnectionFactory rabbitConnectionFactory;
 
+    @Autowired
+    private ErrorHandler errorHandler;
```

Comment:
Yes I need it, if I would use the errorHandler() method the RabbitListenerContainerFactory would always use the default error handler (ConditionalRejectingErrorHandler) and we could not provide a custom one. By autowiring the error handler Spring will take the one in the application context which is either a custom one that is been created or the default one if none has been created.